frontend/javascripts/oxalis/view/action-bar/share_modal_view.tsx

```diff
@@ -153,29 +156,49 @@ function _ShareModalView(props: Props) {
     fetchAndSetSharedTeams();
   }, [annotationType, annotationId, activeUser]);
 
-  const handleCheckboxChange = (event: RadioChangeEvent) => {
-    setVisibility(event.target.value as any as APIAnnotationVisibility);
+  const reportSuccessfulChange = (newVisibility: APIAnnotationVisibility) => {
+    const randomKeyToAllowDuplicates = Math.random().toString(36).substring(0, 5);
```

Comment:
I think you can simply remove the `key` property to get the behavior you want.

Reply:
The key is required as per default the content string of the toast is chosen as a key. Therefore there are no duplicates of the Toast without a manual key. Note that I reduced the timeout time of the toast to 3,5 seconds as 4 secs felt too long for me. Feel free to argue with that :D

Reply:
> The key is required as per default the content string of the toast is chosen as a key.

Are you sure about this? Maybe it changed in the latest antd versions, but the current documentation + example seems to allow duplicate messages if no key is provided. The very first demo here suggests this at least: https://ant.design/components/notification/#header
However, if it doesn't work without the key, the code can obviously stay as is.

> Note that I reduced the timeout time of the toast to 3,5 seconds as 4 secs felt too long for me. Feel free to argue with that :D

Sounds fair :)

Reply:
> Are you sure about this? Maybe it changed in the latest antd versions, but the current documentation + example seems to allow duplicate messages if no key is provided. The very first demo here suggests this at least: https://ant.design/components/notification/#header

Oh, that might be right but we changed this behaviour in our `toast.tsx` lib. Toast code sets a key per default as long as the message of the toast is a plain string.
See: 
https://github.com/scalableminds/webknossos/blob/4e1d4d644e137ee4d06859d43303685ea2211787/frontend/javascripts/libs/toast.tsx#L86

Therefore the explicit key is needed and the key needs to be a random string.

Reply:
ah, I see, thank you for looking this up :+1: